metagraph/CMakeListsKMC.txt.in

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.2)
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
does CMakeListsKMC.txt.in not work with 2.8.2?

Reply:
It was producing warnings that support for version <2.8.12 is deprecated and going to be phased out, so I updated this to suppress those.